services/pull/merge.go

```diff
@@ -34,8 +34,8 @@ import (
 	issue_service "code.gitea.io/gitea/services/issue"
 )
 
-// GetDefaultMergeMessage returns default message used when merging pull request
-func GetDefaultMergeMessage(ctx context.Context, baseGitRepo *git.Repository, pr *issues_model.PullRequest, mergeStyle repo_model.MergeStyle) (message, body string, err error) {
+// Compose message used when merging pull request.
+func getMergeMessage(ctx context.Context, baseGitRepo *git.Repository, pr *issues_model.PullRequest, mergeStyle repo_model.MergeStyle, extraVars map[string]string) (message, body string, err error) {
```

Comment:
A nit (not block), Golang standard (although I do not like it either  ....): the comment starts with function name like:

Otherwise some IDE and lint rules would show warning on it.


```go
// Foo returns bar
func Foo() string {
}
```